packages/youtube-video-element/youtube-video-element.js

```diff
@@ -134,7 +134,10 @@ class YoutubeVideoElement extends (globalThis.HTMLElement ?? class {}) {
           this.isLoaded = true;
           this.loadComplete.resolve();
         },
-        onError: (error) => console.error(error),
+        onError: (error) => {
+          console.error(error);
+          this.dispatchEvent(new CustomEvent('error', { detail: error }));
```

Comment:
I know this can be handy but the native video API doesn't pass any data with the event. I think we should stay 1:1 as much as possible so a `new Event('error')` and then store the last error in a variable and return it in a `get error()` property